### PR TITLE
add clingox/telingo/clorm packages

### PR DESCRIPTION
**Pull request by rkaminsk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@daveraja, please add a confirm that you are willing to (co-)maintain the clorm package

### CODE REVIEW

**Review by daveraja — APPROVED** [review]

**Comment by rkaminsk** [comment]
Hi @conda-forge/help-python, the PR is ready for review.

File: recipes/python-clingox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+source:
+  url: https://github.com/potassco/{{ name }}/archive/v{{ version }}.zip
+  sha256: c947596b38c2f08d39f433a974ea8a9661b9f342f171924eebf2b719dc1d9765
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
```
Comment (dopplershift) [inline recipes/python-clingox/meta.yaml:16]: This can technically just be combined onto the line above.

File: recipes/python-clingox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+requirements:
+  host:
+    - clingo
```
Comment (dopplershift) [inline recipes/python-clingox/meta.yaml:20]: No need for `clingo` in host requirements--only the things needed to run `setup.py` need to be listed.

File: recipes/clorm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - clingo
```
Comment (dopplershift) [inline recipes/clorm/meta.yaml:20]: No need for `clingo` in host requirements.

File: recipes/clorm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home:  https://potassco.org/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```
Comment (dopplershift) [inline recipes/clorm/meta.yaml:39]:
```suggestion
```
This line is both incorrect and superseded just below.

File: recipes/telingo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  noarch: python
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - telingo = telingo:main
+
+requirements:
+  host:
+    - clingo
```
Comment (dopplershift) [inline recipes/telingo/meta.yaml:22]:
```suggestion
```

File: recipes/telingo/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: A solver for temporal logic programs.
+  description: |
+    Telingo is a solver for temporal programs. It leaverages clingo's input
+    language and scripting cababilities to parse and solve programs with
+    temporal formulas. As such the input of telingo is valid clingo input
+    supporting all clingo language features like for example aggregates; only
+    the way programs are grounded and solved is adjusted.
+  license_file: LICENSE.md
```
Comment (dopplershift) [inline recipes/telingo/meta.yaml:48]:
```suggestion
```
Duplicated from above.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Looks in pretty good shape, just a few minor things.

File: recipes/python-clingox/meta.yaml (hunk 2 above)
Comment (rkaminsk) [reply recipes/python-clingox/meta.yaml]: You are right. I was packaging C++ packages before and there it was necessary.

File: recipes/telingo/meta.yaml (hunk 6 above)
Comment (rkaminsk) [reply recipes/telingo/meta.yaml]: It's a pity that the linter does not find this.

Comment (dopplershift) [reply recipes/telingo/meta.yaml]: I *think* (but am by no means sure) that the YAML parsing collapses the duplicates before the linter gets them.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**